Update ingress.md (#8251)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/concepts/ingress.md

---

{:.note}

> > **Note**: Kong will continue to support the [Kubernetes Ingress resource](https://kubernetes.io/docs/concepts/services-networking/ingress/) to configure a {{site.base_gateway}} for the foreseeable future. However, as the [Kubernetes Gateway API resource](https://kubernetes.io/docs/concepts/services-networking/gateway/) is now the preferred mechanism for configure inbound routing in Kubernetes clusters, Kong recommends where possible that you use the Gateway API to configure a {{site.base_gateway}}.

Reviewer: ```suggestion
> **Note**: Kong will continue to support the [Kubernetes Ingress resource](https://kubernetes.io/docs/concepts/services-networking/ingress/) to configure a {{site.base_gateway}} for the foreseeable future. However, as the [Kubernetes Gateway API resource](https://kubernetes.io/docs/concepts/services-networking/gateway/) is now the preferred mechanism for configuring inbound routing in Kubernetes clusters, we recommend that you use the Gateway API to configure a {{site.base_gateway}}.
```
"Where possible" is implied by the recommendation, so we can leave that out.
